web/src/components/inputs/input_wide.tsx

```diff
@@ -5,7 +5,7 @@
 
 import { Field } from "formik";
 import Select from "react-select";
-import { Switch } from "@headlessui/react";
+import { Field as Group, Label, Description } from "@headlessui/react";
```

Comment:
It might make more sense to change the Formik imports in this case to either `* as Formik` and use dot import or `Field as FormikField`.